Assets/HoloToolkit/Input/Scripts/OnGazeEnterEvent.cs

```diff
@@ -1,21 +0,0 @@
-﻿using UnityEngine;
```

Comment:
Haha, that's actually how I started out in Unity. I was just a designer playing with values in the inspector!  Then I got curious... too curious haha!

btw check this out: 
https://forum.unity3d.com/threads/custom-ui-event-system-revamped.353544/

Looks pretty cool. Open source as well.